pkg/cmd/root.go

```diff
@@ -20,6 +20,7 @@ var rootCmd = &cobra.Command{
 
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&helpers.LogLevel, "log-level", "l", "info", helpers.LogLevelMsg)
+	rootCmd.PersistentFlags().BoolVarP(&helpers.ColoredOutput, "color", "C", false, helpers.ColoredOutputMsg)
```

Comment:
I feel like I am ok with not having a short form flag for color. it is a luxury now, so doing `--color` should be just fine. We dont have any other capital short forms for flags, and if we want to introduce one, probably better to do it for an actual feature.

Reply:
I'm ok with that.